src/stories/Library/search-result-page/search-result-page.scss

```diff
@@ -11,3 +11,41 @@
     gap: 16px;
   }
 }
+
+.facet-line {
+  &__item {
+    display: inline-block;
+    @extend %mr-8;
+    @extend %mt-8;
+
+    .dropdown__select {
+      @extend %text-tags;
+      height: 100%;
+      padding: 8px 16px;
+      min-width: 200px;
+    }
+    .dropdown__arrows {
+      height: 100%;
+    }
+  }
+}
+
+.facet-line-selected {
```

Comment:
Mmmm.. okay, so this class is out of place, because "selected" is a modifier to the facet-line class. Please refer to my other comment (from the component file) and adjust these classes.

Reply:
no it is its own block of  markkup and class - it is the band with facets that are selected.

Reply:
We have chosen another name